README.md

```diff
@@ -14,6 +14,13 @@ Create Arma Reforger Dedicated Servers on Windows with this easy to use GUI tool
 ## Requirements
 - [dotNET 6 Runtime](https://dotnet.microsoft.com/en-us/download)
 
+## Discord
+
+Come discuss with fellow users, seek help etc. on the Discord server [here](https://discord.gg/RmTH7gKdfK)
+
+## Settings
```

Comment:
Maybe change to "Server Parameters"

Reply:
Done in https://github.com/soda3x/ArmaReforgerServerTool/pull/50/commits/04f04a46a2926a00fe23faa315bb240ade23a2cf?notification_referrer_id=NT_kwDOAVH0KrQxMDI5OTY4MTU1NjoyMjE0ODEzOA